examples/models/maps_cities.py

```diff
@@ -1,3 +1,12 @@
+''' A plot showing a map of the world, highlighting cities where the population is over 5,000 people, made using the 'GMapPlot' class.
+
+.. bokeh-example-metadata::
+    :sampledata: world_cities
+    :apis: bokeh.models.glyphs.Circle, bokeh.models.map_plots.GMapPlot, bokeh.models.tools.PanTool, bokeh.models.tools.WheelZoomTool
```

Comment:
@Azaya89 at this point we only ever intend to ever show top level `bokeh.models` and avoid any user-facing use of submodules of `bokeh.models`
```suggestion
    :apis: bokeh.models.Circle, bokeh.models.GMapPlot, bokeh.models.PanTool, bokeh.models.WheelZoomTool
```

Reply:
Done